unfoldingWord/translationCore
# Basic Layout and design of ProjectsManagement page. #969

#### This pull request addresses:

Basic Layout and design of ProjectsManagement page.
Temporarily, no real projects from your folder, just using current project twice to simulate list.
No functionality like select or 3dot menu yet.

#### How to test this pull request:

- Use the next button on the homescreen to get to the projects page.
- Open a project from the modal and close modal.
- Review basic layout/design and see if it looks close.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2020)
